packages/experiments/src/components/Card/CardItem/CardItem.styles.ts

```diff
@@ -21,7 +21,18 @@ export const CardItemStyles: ICardItemComponent['styles'] = (props, theme): ICar
       disableChildPadding && {
         marginLeft: -12,
         marginRight: -13
+      },
+      grow && { flexGrow: grow === true ? 1 : grow },
+      !grow &&
+        !shrink && {
+          flexShrink: 0
```

Comment:
Not knowing the internals and just curious... if `Stack` is used why we have all this flexbox specific styles here all over the place.

Reply:
Again, most of this was just copy-pasting things from `Stack.Item`, will probably extend or leverage the `Stack.Item` component as we move forward with this PR.